cli/src/main/java/com/devonfw/tools/ide/tool/GlobalToolCommandlet.java

```diff
@@ -81,7 +81,7 @@ private boolean executePackageManagerCommand(PackageManagerCommand pmCommand, bo
 
     String bashPath = this.context.findBashRequired();
     for (String command : pmCommand.commands()) {
-      ProcessContext pc = this.context.newProcess().errorHandling(ProcessErrorHandling.WARNING).executable(bashPath)
+      ProcessContext pc = this.context.newProcess().errorHandling(ProcessErrorHandling.LOG_WARNING).executable(bashPath)
```

Comment:
I removed this method and IMHO this process is for executing a package manager based installation and does not necessarily share the same logic as a tool execution what is a different thing.
The only thing that I changed here was the name of the `ProcessErrorHandling` enum constants that pointed you to this spot. I would even question if we here should use `LOG_ERROR` instead of WARNING.
I also would say that the code below that checks for the exit code needs improvement:
* it should use the `SUCCESS` constant instead of `0`
* we should log an error if an installation failed.
* we should not return `false` and say that the tool was already installed when it was not and the installation failed. Instead we should throw an exception.
All these kind of issues lead me to this extra large PR but at some point I must stop including everything into this single PR. Feel free to create many new issues for all the mess we might still have.